modules/CMakeLists.txt

```diff
@@ -20,4 +19,5 @@ if (MSVC)
 endif ()
 
 add_subdirectory(image)
+add_subdirectory(reroute)
```

Comment:
Ahh no reason, I suspect this was left over from my (experimental) work in adding the producer code,   Will revert.